### ISSUE TITLE
Integration in mutt-wizard

### ISSUE THREAD

**Issue by gerazov** [anufrievroman/NeomuttFilePicker#4]
https://github.com/LukeSmithxyz/mutt-wizard is a great way to setup neomutt on your system. Since the issue is closed on the  neomutt (https://github.com/neomutt/neomutt/issues/2738) repo, it seems it's not on the roadmap to get integrated there. 

Maybe it would be useful to integrate it into mutt-wizard? :thinking: 

I've submitted an issue there as well https://github.com/LukeSmithxyz/mutt-wizard/issues/937

**Comment by anufrievroman** [comment]
Yeah, as a long-time Luke's subscriber, I would be happy to see it integrated into mutt wizard :D

**Comment by sand4rt** [comment]
related issue: https://github.com/LukeSmithxyz/mutt-wizard/issues/937

**Comment by anufrievroman** [comment]
If somebody would like to put it into Luke's repository, feel free to just copy the scripts there. I mean, there's no point in making dependencies for such a small thing. I don't have this setup currently, so I can't really test I am not familiar with mutt-wizard so much.
